mealie/services/scheduler/scheduler_service.py

```diff
@@ -18,18 +21,43 @@ class SchedulerService:
     @staticmethod
     async def start():
         await run_minutely()
-        await run_daily()
         await run_hourly()
 
+        # Wait to trigger our daily run until our given "daily time", so having asyncio handle it.
+        asyncio.create_task(schedule_daily())
+
+
+async def schedule_daily():
+    now = datetime.now()
+    daily_schedule_time = get_app_settings().DAILY_SCHEDULE_TIME
+    logger.debug(
+        "Current time is %s and DAILY_SCHEDULE_TIME is %s",
+        str(now),
+        daily_schedule_time,
+    )
+    hour_target = int(daily_schedule_time.split(":")[0])
+    minute_target = int(daily_schedule_time.split(":")[1])
+
+    hours_until = ((hour_target - now.hour) % 24) or 24
+    minutes_until = (minute_target - now.minute) % 60
+    logger.debug("Hours until %s and minutes until %s", str(hours_until), str(minutes_until))
+
+    delta = timedelta(hours=hours_until, minutes=minutes_until)
+    target_time = (now + delta).replace(microsecond=0, second=0)
+    logger.info("Daily tasks scheduled for %s", str(target_time))
+    wait_seconds = (target_time - now).total_seconds()
+    await asyncio.sleep(wait_seconds)
```

Comment:
Here we sleep to the first scheduled time, which is why we don't want to "wait" on the first run_daily - this is where that happens.